### PR TITLE
feat: add thumbnail information to event

### PR DESCRIPTION
**Pull request by fampinheiro**
https://trello.com/c/C4S4plSK/1011-event-thumbnails

### CODE REVIEW

File: packages/react-components/src/molecules/EventInfo.tsx [hunk 1]
```diff
@@ -54,6 +54,12 @@ const eventOwnerStyles = css({
   textOverflow: 'ellipsis',
 });
 
+const imageStyle = css({
+  objectFit: 'cover',
```
Comment (jeysal) [inline packages/react-components/src/molecules/EventInfo.tsx:58]: checked w/ design?

File: packages/react-components/src/molecules/EventInfo.tsx [hunk 2]
```diff
@@ -83,9 +93,15 @@ const EventInfo: React.FC<EventInfoProps> = ({
     'E, d MMM y',
   ).toUpperCase();
 
+  const imageComponent = thumbnail ? (
+    <img alt={`"${title}"'s thumbnail`} src={thumbnail} css={[imageStyle]} />
```
Comment (jeysal) [inline packages/react-components/src/molecules/EventInfo.tsx:97]:
```suggestion
    <img alt={`Thumbnail for "${title}"`} src={thumbnail} css={imageStyle} />
```

File: packages/react-components/src/molecules/__tests__/EventInfo.test.tsx [hunk 3]
```diff
@@ -29,6 +29,28 @@ it('truncates long event titles', () => {
...
+  );
+
+  expect(getByAltText(/thumbnail/i).getAttribute('src')).toEqual(
+    'https://placeholder/40x40',
+  );
+});
+
+it('renders placeholder event thumbnail', () => {
+  const { getByText } = render(
+    <EventInfo {...props} title={'blablablha'.repeat(100)} />,
```
Comment (jeysal) [inline packages/react-components/src/molecules/__tests__/EventInfo.test.tsx:48]: explicit `thumbnail={undefined}` - we don't know what the example data looks like

**Review by jeysal — CHANGES_REQUESTED** [review]
some small suggestions, but can only speak from FE POV ofc :D

File: packages/react-components/src/molecules/__tests__/EventInfo.test.tsx (hunk 3 above)
Comment (fampinheiro) [reply packages/react-components/src/molecules/__tests__/EventInfo.test.tsx:48]:
```suggestion
    <EventInfo {...props} thumbnail={undefined} title={'blablablha'.repeat(100)} />,
```

**Review by johnytiago — APPROVED** [review]

File: packages/react-components/src/molecules/EventInfo.tsx (hunk 1 above)
Comment (fampinheiro) [reply packages/react-components/src/molecules/EventInfo.tsx:58]: https://trello.com/c/C4S4plSK/1011-event-thumbnails#comment-603f64ea04f11a66611d56cc

**Review by jeysal — APPROVED** [review]

**Pull request merged**